course05/homework01/stack_string/stack_string.cc

```diff
@@ -0,0 +1,58 @@
+#include "stack_string.h"
+
+#include <iostream>
+
+namespace cppcourse::homework01 {
+
+StackString::StackString(const std::string& input)
+{
+    if (input.length() > MAX_STRING_LENGTH)
+		throw std::runtime_error("String length exceeds MAX_STRING_LENGTH (" + std::to_string(MAX_STRING_LENGTH) + ")");
+
+    mTop = input.length();
+	strcpy(mData, input.c_str());
+	mData[mTop+1] = '\0';
```

Comment:
Is this correct? is the size is 1, then you should set the null character at `[1]`, not `[2]`?